src/core/ext/transport/binder/utils/transport_stream_receiver_impl.h

```diff
@@ -50,12 +50,17 @@ class TransportStreamReceiverImpl : public TransportStreamReceiver {
                                   absl::StatusOr<Metadata> trailing_metadata,
                                   int status) override;
 
-  void CancelRecvMessageCallbacksDueToTrailingMetadata(
-      StreamIdentifier id) override;
-  void Clear(StreamIdentifier id) override;
-  void CancelStream(StreamIdentifier, absl::Status error) override;
+  void CancelStream(StreamIdentifier id) override;
 
  private:
+  // Trailing metadata marks the end of one-side of the stream. Thus, after
+  // receiving trailing metadata from the other-end, we know that there will
+  // never be in-coming message data anymore, and all recv_message callbacks
+  // (as well as recv_initial_metadata callback, if there's any) registered will
+  // never be satisfied. This function cancels all such callbacks gracefully
+  // (with GRPC_ERROR_NONE) to avoid being blocked waiting for them.
```

Comment:
Wrong comment location?

Reply:
I think this is the place it should be. Can you elaborate why it's at the wrong location?